site/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    .addEntry('app', ['./web/assets/scss/librecores.scss',
+        './web/assets/css/bootstrap-social.css',
+        './web/assets/css/font-awesome.min.css',
+        './web/assets/css/librecores.css',
+        './web/assets/scss/bootstrap-librecores.scss'],
+        )
+    //.addEntry('page1', './assets/js/page1.js')
+    //.addEntry('page2', './assets/js/page2.js')
+
+    // will require an extra script tag for runtime.js
+    // but, you probably want this, unless you're building a single-page app
+    .enableSingleRuntimeChunk()
+
+    .cleanupOutputBeforeBuild()
+    .enableSourceMaps(!Encore.isProduction())
+    // enables hashed filenames (e.g. app.abc123.css)
+    .enableVersioning(Encore.isProduction())
+
+// uncomment if you use TypeScript
+//.enableTypeScriptLoader()
+
+// uncomment if you use Sass/SCSS files
+  .enableSassLoader()
+
+// uncomment if you're having problems with a jQuery plugin
```

Comment:
Do we have problems with jQuery? If so, what problems do we have?

Reply:
I think we may have. Most of the jQuery plugins we use (Select2, Algolia) are configured to use jQuery as a global than require it.

Reply:
So please mention this fact in the comment.